frontend/src/scenes/session-recordings/filters/SessionRecordingsFilters.tsx

```diff
@@ -41,6 +48,51 @@ const filtersToLocalFilters = (filters: RecordingFilters): LocalRecordingFilters
     }
 }
 
+function ConsoleFilters({
+    filters,
+    setConsoleFilters,
+}: {
+    filters: RecordingFilters
+    setConsoleFilters: (selection: FilterableLogLevel[]) => void
+}): JSX.Element {
+    function updateChoice(checked: boolean, level: FilterableLogLevel): void {
+        const newChoice = filters.console_logs?.filter((c) => c !== level) || []
+        if (checked) {
+            setConsoleFilters([...newChoice, level])
+        } else {
+            setConsoleFilters(newChoice)
+        }
+    }
+
+    return (
+        <div className="flex items-center gap-1 flex-wrap font-medium text-primary-alt" data-attr={'console-filters'}>
+            <LemonSwitch
+                size="small"
+                bordered={true}
+                checked={!!filters.console_logs?.includes('log')}
+                onChange={(checked) => {
+                    updateChoice(checked, 'log')
+                }}
+                label={'include console.log'}
+            />
+            <LemonSwitch
+                size="small"
+                bordered={true}
+                checked={!!filters.console_logs?.includes('warn')}
+                onChange={(checked) => updateChoice(checked, 'warn')}
+                label={'include console.warn'}
+            />
+            <LemonSwitch
+                size="small"
+                bordered={true}
+                checked={!!filters.console_logs?.includes('error')}
+                onChange={(checked) => updateChoice(checked, 'error')}
+                label={'include console.error'}
+            />
+        </div>
```

Comment:
We _could_ use a multi select checkbox, like we do for the paths page. Feels a little more intuitive...
<img width="354" alt="Screenshot 2023-06-09 at 08 21 02" src="https://github.com/PostHog/posthog/assets/2536520/7b78bad1-8a28-46dc-ae98-2c92be6fb079">